Outcomes/outcomes.py

```diff
@@ -62,18 +66,19 @@ def onerun_delayframe_outcomes_load_hpar(config, in_run_id, in_prefix, in_sim_id
     )).to_pandas()
 
     if npi_config is not None:
-        diffI, places, dates = read_seir_sim(in_run_id, in_prefix, in_sim_id)
-        npi = NPI.NPIBase.execute(
-            npi_config=npi_config[0],
-            global_config=npi_config[1],
-            geoids=places,
-            loaded_df = npi_load(
-                file_paths.create_file_name_without_extension(
-                    in_run_id,
-                    in_prefix, 
-                    in_sim_id,
-                    "hnpi"),'parquet')
-        )
+        with Timer('onerun_delayframe_outcomes_load_hpar.NPI'):
```

Comment:
You're leaving in the Timer calls?

Reply:
yes, they're only print something depending on logging level, and don't take a lot of time, if that's ok